cpp/tests/test_abm_world.cpp

```diff
@@ -302,19 +302,17 @@ TEST(TestWorld, evolveMigration)
         auto t      = mio::abm::TimePoint(0) + mio::abm::hours(1);
         auto dt     = mio::abm::hours(2);
         auto params = mio::abm::GlobalInfectionParameters{};
-        params.get<mio::abm::SevereToCritical>()[{mio::abm::VirusVariant::Wildtype, mio::abm::AgeGroup::Age60to79,
-                                                  mio::abm::VaccinationState::Unvaccinated}] = 0.01;
         params.get<mio::abm::CriticalToDead>()[{mio::abm::VirusVariant::Wildtype, mio::abm::AgeGroup::Age60to79,
-                                                mio::abm::VaccinationState::Unvaccinated}]   = 0.01;
+                                                mio::abm::VaccinationState::Unvaccinated}] = 0.01;
```

Comment:
I think a viable option is:

`SevereToCritical = dt`
`CriticalToDead = 0.5*dt`

Reply:
Yes, your solution works perfectly for me. As SevereToCritical+CriticalToDead > dt, the Person only has time to go back to InfectedSevere at t. 
I also adjusted the t and dt to days to make it easier to understand.